assessments/KRAssessment/templates/assessments.jsont

```diff
@@ -0,0 +1,19 @@
+{
+  "assessmentIdentifier": "{{assessmentIdentifier}}",
+  "assessmentTitle": "{{assessmentTitle}}",
+  "namespace": "{{namespace}}",
+  "assessedGradeLevels": [{{grade_json}}],
+  "assessmentCategoryDescriptor": "uri://ed-fi.org/assessmentCategoryDescriptor#{{assessmentCategoryDescriptor}}",
```

Comment:
may be worth just including the entire namespace in the seed so it doesn't need to be hardcoded here